opensrp-anc/src/main/java/org/smartregister/anc/library/activity/MainContactActivity.java

```diff
@@ -657,6 +660,15 @@ private void updateDefaultValues(JSONArray stepArray, int i, JSONObject fieldObj
                 }
             }
         }
+
+        if (fieldObject.getString(JsonFormConstants.KEY).equals("record_bp_using_optibp_button")
+                || fieldObject.getString(JsonFormConstants.KEY).equals("record_bp_using_optibp_2nd_reading_button")) {
```

Comment:
Let's move the hardcoded strings to the constants files

Reply:
Done